### PR TITLE
Fix typo, and removed pytest-runner

### PR DESCRIPTION
**Pull request by mrunge**
pytest-runner is not required for install itself.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -33,12 +33,11 @@
     keywords='SQLAlchemy collectd',
     author='Mike Bayer',
     author_email='mike@zzzcomputing.com',
-    url='http://github.org/sqlalchemy/sqlalchemy-collectd',
+    url='https://github.com/sqlalchemy/sqlalchemy-collectd',
     license='MIT',
     packages=find_packages(".", exclude=["*.tests"]),
     include_package_data=True,
     tests_require=['pytest', 'mock'],
-    setup_requires=['pytest-runner'],
```
Comment (zzzeek) [inline setup.py]: this is so you can run "python setup.py test".    if we take this out then we have to pick a different test runner which means we'd need to change "test=pytest" inside of setup.cfg and probably some other things.

**Review by zzzeek — COMMENTED** [review]
it looks like the issue here is that pytest-runner's default approach is to include it in setup.     I'd like python setup.py test to still work somehow so what I do everywhere else is https://docs.pytest.org/en/latest/goodpractices.html#manual-integration , e.g. make a separate setuptools command without using pytest-runner.   I guess we can use unittest also but I don't know how to get unittest to run with the way I've placed the test files here.

File: setup.py (hunk 1 above)
Comment (mrunge) [reply setup.py]: In a quick test, I failed to follow the goodpractices manual to by adding a additional class and to make it work. Maybe you'd have the time to look at it?

**Comment by mrunge** [comment]
So, unfortunately, there is no pytest-runner currently on CentOS build system.
The tests are not distributed via pypi anyways, that makes it a bit awkward to pull  in test dependencies and to not execute tests then.

**Comment by mrunge** [comment]
Nevertheless, just by removing setup_requires, I still can execute tests (they are successful), and I could build the package without needing pytest-runner. What do you think?

**Comment by zzzeek** [comment]
I definitely will take out pytest-runner as this gives me a good reason to not have to worry about that thing again.   I'll put in the manual approach which is what I do everywhere else, e.g. https://github.com/zzzeek/alembic/blob/master/setup.py#L28 and you'll never have to deal with it.   ill close this PR once I push that in later today.   does that work ?  thanks for looking.

**Comment by mrunge** [comment]
Thank you for looking at it, it really helps me in packaging it.

**Comment by zzzeek** [comment]
ok see 94f0906a1c3bd59d3b8ba7ded714e2447d95c58c for pytest-runner and 5b0a208742f6aba9e639b3144d6f0b2c24018139 for the correct github link :)   will the .spec be running from pypi?  let me know when I can put in a new version.

**Pull request closed without merging**